src/coreclr/nativeaot/Runtime/RestrictedCallouts.h

```diff
@@ -98,5 +98,5 @@ class RestrictedCallouts
 
     // Prototypes for the callouts.
     typedef void (REDHAWK_CALLCONV * GcRestrictedCallbackFunction)(uint32_t uiCondemnedGeneration);
-    typedef CLR_BOOL (REDHAWK_CALLCONV * HandleTableRestrictedCallbackFunction)(Object * pObject);
+    typedef CLR_BOOL (STDCALL * HandleTableRestrictedCallbackFunction)(Object * pObject);
```

Comment:
```suggestion
    typedef CLR_BOOL (*HandleTableRestrictedCallbackFunction)(Object * pObject);
```
Use default

Reply:
Don't all compilers default to `CDECL` even on Windows?

Reply:
We set the default to stdcall on Windows x86.